tests/tests/test_mender_connect.py

```diff
@@ -23,8 +23,8 @@
 from testutils.infra.container_manager.kubernetes_manager import isK8S
 from testutils.infra.device import MenderDevice
 from ..common_setup import (
-    class_persistent_standard_setup_one_client_bootstrapped,
-    enterprise_no_client_class,
+    class_persistent_standard_setup_one_client_bootstrapped_rt,
+    enterprise_no_client_class_rt,
```

Comment:
This is the exact same as 
```suggestion
    standard_setup_one_client_bootstrapped,
    enterprise_no_client,
```